README.md

```diff
@@ -91,6 +92,32 @@ bwrap --ro-bind /usr /usr \
    /bin/sh
 ```
 
+Sandboxing
+----------
+
+The goal of bubblewrap is to run an application in a sandbox, where it has
+access to less resources.
```

Comment:
s/access to less resources/restricted access to parts of the operating system or user data such as the home directory/ ?